Sources/LiveKit/Views/VideoView.swift

```diff
@@ -303,10 +300,19 @@ private extension VideoView {
 
         if shouldAttach {
             log("Renderer: attaching...")
+
+            if let frame = track.videoFrame {
+                log("rendering cached frame")
+                renderFrame(frame)
```

Comment:
Will this cause a double effect kind of thing on re-attach? Maybe, it is blank. So, when scrolled into view, it goes from blank -> cached frame (frozen video of last frame) -> jumps ahead to live video?

Reply:
logically, that could happen, but in reality it seems to be instantaneously rendered.

https://user-images.githubusercontent.com/548776/161813898-aadd1ef2-700a-4f1e-90d4-3ad452583a3e.mov